modules/n1ql/pages/n1ql-language-reference/alterindex.adoc

```diff
@@ -296,6 +296,8 @@ nodes;;
 If `action` is set to `move`, the node list determines the new destination nodes for the index and its replicas.
 If `action` is set to `replica_count` and you are _increasing_ the number of replicas, the node list restricts the set of nodes available for placement of the index and its replicas.
 However, if `action` is set to `replica_count` and you are _decreasing_ the number of replicas, the `nodes` property is ignored.
++
+include::partial$n1ql-language-reference/file-based-index-rebalance-note.adoc[]
```

Comment:
This restriction is only applicable for "action" set to "move". Other actions are allowed.

Reply:
Just to clarify: do you mean the restriction on using nodes when file-based indexing is enabled is limited to the move action? You can use nodes with index rebalancing enabled for replica_count and drop_replica?

Reply:
I think I misread the documentation. It is correct. The "nodes" will be ignored for "replica_count" as well when file-based rebalancing is enabled.